composer.json

```diff
@@ -22,16 +22,35 @@
       "email": "thomas.ploch@flixbus.com"
     }
   ],
+  "repositories": [
+    {
+      "type": "vcs",
+      "url": "https://github.com/MyOnlineStore/avro-php"
+    },
+    {
+      "type": "vcs",
+      "url": "https://github.com/MyOnlineStore/fantasy-land"
+    },
+    {
+      "type": "vcs",
+      "url": "https://github.com/MyOnlineStore/php-functional"
+    },
+    {
+      "type": "vcs",
+      "url": "https://github.com/MyOnlineStore/schema-registry-php-client"
+    }
+  ],
   "require": {
-    "php": "^7.1",
+    "php": "^7.0",
     "beberlei/assert": "~2.7",
-    "flix-tech/confluent-schema-registry-api": "^6.0.0",
-    "flix-tech/avro-php": "^3.0.0",
-    "widmogrod/php-functional": "^4.2|^5.0"
+    "flix-tech/confluent-schema-registry-api": "dev-backport-7.0",
+    "flix-tech/avro-php": "dev-backport-7.0",
+    "functional-php/fantasy-land": "dev-backport-7.0",
```

Comment:
Why did you add this? It wasn't in the original list of requirements.

Reply:
This was needed for Composer to be able to resolve the `dev-backport7.0` reference.